scripts/setup.sh

```diff
@@ -186,6 +186,9 @@ if [ ! -d "$NODE_PATH" ]; then
   chmod -R 744 $NODE_MODULE_DIR
 fi
 
+# Change execute status of browsermob-proxy
+chmod +x $TOOLS_DIR/browsermob-proxy-2.1.1/bin/browsermob-proxy
```

Comment:
Yes, the node module is also js bindings for browsermob-proxy (binary), so we'll need to have the binary which would need the executable bit flipped.

One workaround is, zipping the browsermob-proxy folder after the execute bit had been set. When we unzip it, the execute bit should remain set.